js/todos.js

```diff
@@ -0,0 +1,37 @@
+var App = new (Backbone.View.extend({
+    Models: {},
+    Views: {},
+    Collections: {},
```

Comment:
I think in general I'd be tempted to separate these. So, you'd have an App View that was your App and it would have the collection, but you'd just have some general JS that would boot everything up.

There's a tendency in Backbone to want to make everything a View, Model, or Collection, but sometimes code is just code, and you get a testing/maintenance advantage for decoupling them. In this case you're really only coupling your AppView to the TodoCollection, and your code can reflect that.